hackerank/js-practice/function.js

```diff
@@ -0,0 +1,8 @@
+function factorial(n) {
+	if (n < 1) {
+		return 1;
+	} else if (n < 10) {
```

Comment:
Hi @EleonoraDellamico , why do you  add this `} else if (n < 10) {` ? It's a requirement of the practice, or you are playing with recursion?